core/client/pom.xml

```diff
@@ -49,15 +54,20 @@
       <artifactId>slf4j-api</artifactId>
     </dependency>
 
-    <!-- Other Alluxio dependencies -->
+    <!-- Internal dependencies -->
     <dependency>
       <groupId>org.alluxio</groupId>
-        <artifactId>alluxio-core-protobuf</artifactId>
-        <version>${project.version}</version>
-      </dependency>
+      <artifactId>alluxio-core-common</artifactId>
+      <version>${project.version}</version>
+    </dependency>
     <dependency>
       <groupId>org.alluxio</groupId>
-      <artifactId>alluxio-core-common</artifactId>
+      <artifactId>alluxio-core-protobuf</artifactId>
```

Comment:
as per discussion with @calvinjia and @apc999 ... we want to include all direct dependencies instead of relying on transitive ones ... I will have a follow up PR that enforces this